osm2lanes/src/locale.rs

```diff
@@ -73,6 +73,28 @@ impl Locale {
         }
     }
 
+    /// Highway type with no `lanes=*` has one bidirectional lanes or two directions lanes
```

Comment:
Having trouble parsing this. "has one lane in each direction"? "one bidirectional lane" to me means "1 lane, that can go both directions"